unlock-app/package.json

```diff
@@ -93,27 +93,20 @@
     "start-ganache:win32": "cd .. && (START /b npm run start-ganache -- -b 3 ) ",
     "test": "cross-env UNLOCK_ENV=test jest --env=jsdom",
     "lint": "eslint .",
-    "reformat": "prettier-eslint \"src/**/*.js\" --write",
-    "fail-pending-changes": "../scripts/pending-changes.sh",
     "storybook": "start-storybook -p 9001 -c .storybook -s src",
     "chromatic": "chromatic test --exit-zero-on-changes",
     "svg-2-components": "./node_modules/@svgr/cli/bin/svgr --title-prop --no-dimensions --template src/components/interface/svg/template.js --no-dimensions -d src/components/interface/svg/ src/static/images/svg/",
-    "eslint-check": "eslint --print-config . | eslint-config-prettier-check",
-    "ci": "npm run lint && npm test && npm run reformat && npm run fail-pending-changes",
+    "ci": "npm run lint && npm test",
     "depcheck": "depcheck --parsers=\"*.js:react,*.js:es6\"",
     "npm-check": "npm-check -u",
     "tsc": "tsc"
   },
   "lint-staged": {
     "linters": {
-      "*.{js,ts,tsx}": [
-        "prettier-eslint --write",
-        "git add"
+      "*.js": [
```

Comment:
Do we only want to run this on the `.js` files?

Reply:
ha! probably not, good catch!